src/utils/graph/solver.js

```diff
@@ -0,0 +1,146 @@
+import * as kiwi from 'kiwi.js';
```

Comment:
So I just looked up kiwi.js and the [full library](https://github.com/gamelab/kiwi.js/blob/master/build/kiwi.min.js) is 322kb minified. Guess it's not possible to just pick and choose what we need, huh? I ask only because if/when we web-workerize this layout function, any libraries imported will need to be bundled into the web worker.

Reply:
Oh that's not the right `kiwi.js` actually, it's [this one](https://github.com/IjzerenHein/kiwi.js/), which seems to be about 14kb minified which I think is fine?

Reply:
ahhhhh phew. Ok that should be sweet then, cheers 👍